### ISSUE TITLE
Cloning dashboards failed with error saying the original dashboard can't be open

### ISSUE THREAD

**Issue by sol1-matt** [meerkat-dashboard/meerkat#207]
**Describe the bug**
Cloning dashboards results in a error 
```
create dashboard from dashboards/test01.json: create dashboard file: open dashboards/test01.json: permission denied
```

**To Reproduce**
1. Clone dashboard test01 to test02
2. See error

**Comment by sol1-matt** [comment]
This isn't fixed in 3.0.4

**Comment by sol1-matt** [comment]
New problem with cloning.

The cloned filename is similar to the title (spaces replaced with dashes) but meerkat is expecting an all lowercase version of the title.

**Solution**
Clone should use the same function that sanitized the filename that create or rename does.
